.spelling

```diff
@@ -125,3 +125,10 @@ Utility.app
 rc
 nginx.conf
 VirtualBox
+proxied
+Frontpage
+anymore
+analyze
```

Comment:
What's wrong with `anymore` and `analyze`? I mean, why do we need to add them in here in first place?